packages/common-library/src/common_library/basic_types.py

```diff
@@ -1,5 +1,12 @@
 from enum import StrEnum
 
+from pydantic_core import PydanticUndefined
+
+# SEE https://github.com/fastapi/fastapi/blob/master/fastapi/_compat.py#L75-L78
+#
+# Use as default when default_factory
+Undefined = PydanticUndefined
```

Comment:
why you do not make it Any here actually?

Reply:
@sanderegg I followed the approach by fastapi.